### PR TITLE
Add speech biasing hints setting method

### PR DESCRIPTION
**Pull request by chilno**
Added a method for setting [speech biasing hints](https://developers.google.com/actions/reference/rest/Shared.Types/AppResponse#ExpectedInput) in the response JSON. It takes an array as a parameter.

### CODE REVIEW

File: actions-sdk-app.js [hunk 1]
```diff
@@ -321,6 +321,55 @@ class ActionsSdkApp extends AssistantApp {
   }
 
   /**
+   * Set speech biasing hints for the conversation lifetime.
+   *
+   * @example
+   * const app = new App({request: req, response: res});
+   * const SPEECH_BIASISNG_HINTS = ['word1', 'word2', 'word3'];
```
Comment (joelhegg) [inline actions-sdk-app.js]: Typo in `BIASING` here and below.

**Comment by chilno** [comment]
Added tests

**Comment by chilno** [comment]
Thanks for the feedback! I understand your concern. I do not think extending the buildInputPrompt() method is going to help in anyway as the speechBiasingHints key is expected in the [ExpectedInput](https://developers.google.com/actions/reference/rest/Shared.Types/AppResponse#ExpectedInput) object.
The reason I used this approach is because I followed what has been done for the setContext() method of the dialogflow-app.js

**Comment by Canain** [comment]
Now that [v2 is released](https://developers.google.com/actions/reference/nodejs/lib-v1-migration), setters like this make more sense as the library more builds the response for the developer.

Feel free to open another PR with something like this in the v2 library. If you want, you can also create a [middleware](https://developers.google.com/actions/reference/nodejs/lib-v1-migration#scaling_with_plugins_and_middlewares) that adds this functionality temporary or as a plugin for yourself or others while it's not in the library.

**Pull request closed without merging**